doc/sphinx/user-extensions/proof-schemes.rst

```diff
@@ -8,35 +8,36 @@ Proof schemes
 Generation of induction principles with ``Scheme``
 --------------------------------------------------------
 
-The ``Scheme`` command is a high-level tool for generating automatically
-(possibly mutual) induction principles for given types and sorts. Its
-syntax follows the schema:
+.. cmd:: Scheme {? @ident := } @scheme_kind {* with {? @ident := } @scheme_kind }
 
-.. cmd:: Scheme @ident__1 := Induction for @ident__2 Sort @sort {* with @ident__i := Induction for @ident__j Sort @sort}
+   .. insertprodn scheme_kind sort_family
 
-  This command is a high-level tool for generating automatically
+   .. prodn::
+      scheme_kind ::= Equality for @reference
+      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
+      sort_family ::= Set
+      | Prop
+      | SProp
+      | Type
+
+  A high-level tool for automatically generating
   (possibly mutual) induction principles for given types and sorts.
-  Each :n:`@ident__j` is a different inductive type identifier belonging to
+  Each :n:`@ident` is a different inductive type identifier belonging to
```

Comment:
That's `@reference` here. It seems that you have marked this as resolved without fixing it.

Reply:
It's so hard to find good workers these days, especially with the low pay.  :-)